src/test/java/seedu/address/logic/commands/RestoreBackupCommandTest.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.commands;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.UndoRedoStack;
+import seedu.address.model.AddressBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.UserPrefs;
+import seedu.address.storage.AddressBookStorage;
+import seedu.address.storage.JsonUserPrefsStorage;
+import seedu.address.storage.StorageManager;
+import seedu.address.storage.XmlAddressBookStorage;
+
+public class RestoreBackupCommandTest {
+    @Rule
+    public TemporaryFolder testFolder = new TemporaryFolder();
+
+    private Model model;
+    private UserPrefs userPrefs;
+    private StorageManager storageManager;
+    private AddressBookStorage addressBookStorage;
+    private String filePath;
+    private String backupFilePath;
+    private RestoreBackupCommand restoreBackupCommand;
+
+    @Before
+    public void setUp() throws Exception {
+        filePath = testFolder.getRoot().getPath() + "TempAddressBook.xml";
```

Comment:
"TempAddressBook.xml" Can there be a final String that takes this value? and invoke the string in Backup and Restore Backup?
It is a bit of magic number here.
It is easier to maintain in the future if you don't want to call it TempAddressBook.xml

Reply:
Same for "TempAddressBook-backup.xml"